ark/type/keywords/string.ts

```diff
@@ -70,6 +70,25 @@ export declare namespace stringInteger {
 	}
 }
 
+const hex = Scope.module(
```

Comment:
Since this doesn't have any subtypes, it should just b a keyword directly. See `string.email` as a good example of where it should go.